src/controls/qml/Application.qml

```diff
@@ -47,13 +48,15 @@ Application_p {
         edge: Qt.RightEdge
         visible: false
         z: 10
+        anchors.verticalCenterOffset: edge === Qt.LeftEdge || edge === Qt.RightEdge ? DeviceInfo.flatTireHeight/2 : 0
```

Comment:
If we move this logic directly in Indicator.qml and guard it with a boolean property, something like compensateForFlatTire then we can still give control to the user, they don't have to think about the maths and we can provide a sane default (I think the sane default is to compensate for the flat tire but if the user really has a reason to think otherwise then they could disable it with the right boolean

Reply:
As I mentioned in the first comment, I think these things should be handled on a case-by-case basis. I spent a few days trying to come up with a neat, universal way of compensating for flat tyres by adding new anchors to qml-asteroid, but I came to the conclusion that these issues should be handled individually by developers in app code. 
I did try putting this logic in the indicators rather than in the window, but this also affected the indicators in the launcher, which are already at the right height. 
I don't think it's worth adding a boolean for this. The indicators are currently used in only two places: the asteroid window (application) item and the launcher, and any further uses will likely be quite unique, so custom positioning will be needed anyway.